src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1775,7 +1775,12 @@ private function export($value)
 
     private function doExport($value, $resolveEnv = false)
     {
-        if (is_string($value) && false !== strpos($value, "\n")) {
+        if (is_string($value) && preg_match('/[\x00-\x08\x0b-\x1f\x7f-\xff]+/', $value,$matches)) {
```

Comment:
Is this really necessary?

Reply:
Yes, otherwise, PhpDumper generates invalid UTF-8 files. As I was unable to properly add safely required characters